frontend/src/routes/(authenticated)/admin/EditUserAccount.svelte

```diff
@@ -16,15 +16,23 @@
   export let currUser: LexAuthUser;
   export let deleteUser: (user: User) => void;
 
-  const schema = z.object({
+  const schema = z
+    .object({
     email: z.string().email($t('form.invalid_email')).nullish(),
     name: z.string(),
     password: passwordFormRules($t).or(emptyString()).default(''),
     score: z.number(),
     role: z.enum([UserRole.User, UserRole.Admin]),
-  });
+    })
+  const refinedSchema = schema
```

Comment:
I think you can just call refine after object then there's no need for the `refinedSchema` variable at all.

Reply:
That is what I initially did, but @rmunn separated them because of some Zod type error.

Reply:
See commit https://github.com/sillsdev/languageforge-lexbox/pull/892/commits/4085759281f285be29e6ef65ce06c3bd56543c4b for why I did this (and what exactly I did). It might be possible to change our Typescript type definitions for FormModal (which relies on types from Zod), but I took one look at the number of changes that would have involved and said, "Nope, we're going to do this the easy way."